src/main/resources/log4j.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+    <appender name="console" class="org.apache.log4j.ConsoleAppender">
+        <param name="Target" value="System.out"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%-5p %c{1} - %m%n"/>
+        </layout>
+        <filter class="org.apache.log4j.varia.LevelRangeFilter">
+            <param name="levelMin" value="WARN" />
+            <param name="levelMax" value="FATAL" />
+        </filter>
+    </appender>
+
+    <root>
+        <priority value ="debug" />
+        <appender-ref ref="console" />
+    </root>
+
+</log4j:configuration>
```

Comment:
Well, I just chose what I used more than a year ago. The XML file is small anyway. But I'll check the other choice tomorrow.